app/controllers/katello/api/v2/content_view_filters_controller.rb

```diff
@@ -82,6 +82,49 @@ def destroy
       respond_for_show :resource => @filter
     end
 
+    api :put, "/content_views/:content_view_id/filters/:id/remove_filter_rules", N_("bulk delete filter rules")
+    api :put, "/content_view_filters/:id/remove_filter_rules", N_("bulk delete filter rules")
+    param :content_view_id, :number, :desc => N_("content view identifier")
+    param :id, :number, :desc => N_("filter identifier"), :required => true
+    param :rule_ids, Array, of: :number, :desc => N_("filter identifiers"), :required => true
+    def remove_filter_rules
+      rule_clazz = ContentViewFilter.rule_class_for(@filter)
+      rule_clazz.where(id: params[:rule_ids]).try(:destroy_all)
+      respond_for_show(resource: @filter)
+    end
+
+    api :put, "/content_views/:content_view_id/filters/:id/add_filter_rules", N_("bulk add filter rules")
+    api :put, "/content_view_filters/:id/add_filter_rules", N_("bulk add filter rules")
+    param :content_view_id, :number, :desc => N_("content view identifier")
+    param :id, :number, :desc => N_("filter identifier"), :required => true
+    param :rules_params, Array, :desc => N_("Rules to be added"), :required => false
```

Comment:
we should probably include the params that this accepts, similar to this:

```
    param :uploads, Array, :desc => N_("Array of uploads to import") do
      param 'id', String, :required => true
      param 'content_unit_id', String
      param 'size', String
      param 'checksum', String
      param 'name', String, :desc => N_("Needs to only be set for file repositories or docker tags")
      param 'digest', String, :desc => N_("Needs to only be set for docker tags")
    end
```

Reply:
Updated! 👍🏽